### ISSUE TITLE
Content support for Supplemental Claims PDF confirmation

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#61023]
## Issue Description

The benefits decision review team opened an intake request asking for support and guidance with 2 pieces of content:

- Button language for confirmation page
- Confirmation PDF memo language

Intake request: #60892

Supporting artifacts:

- [Confirmation Page Design ](https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/a/eldM8Zx)
- [Mockup of Confirmation PDF](https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/prototype/a/68D11FB7-4DFF-4282-9872-3A6DF9E279F0)
- Link to any epics/issues that may be helpful: https://github.com/department-of-veterans-affairs/va.gov-team/issues/55397

## Tasks
- [x] Review content (Katherine)
- [x] OCTO review (Beth)
- [ ] Provided product team with final content (Katherine)

## Acceptance Criteria
- [ ] Product team has final content

**Comment by katherine-fung** [comment]
Hi @bethpottsVADEPO, 

I've attached the draft copy of the Supplemental Claims confirmation memo for your review. 

For context, Eileen let me know: 

- The product team wants to let the user take one action to get both the confirmation page and a copy of their completed application together. They say providing both documents together will give the user context for each document, and reduce the chances that someone will try to mail in a printout of their PDF application. 

- The product team wants to give the user the opportunity to download, rather than print (the rationale is that once users have the downloaded file, they can print if they want to). They're leaning towards the action being an automatic download, similar to VA benefit letters. Eileen said according to the team's accessibility specialist, this automatic download action would have to be a link - not a button. 

Other notes: 
- I added two H3 sections to the confirmation page: 1. "Confirmation for your records" that gives a few more details about downloading your completed application for your records, and 2. "How to contact us if you have questions" with contact info, and how to request a hearing about a Supplemental Claim. (I provided more information about a Veteran's legal right to a hearing in the Word comments)  

- The Veteran will download the confirmation page, and a copy of their completed application which will have a watermark. I suggest some alternate text for the watermark in the Word doc. 

Here's the Sketch link for the confirmation page and application: 
https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/p/C0718005-83F4-4194-B62B-B63FC5F05282/canvas

Here's the Word doc: [https://app.zenhub.com/files/133843125/be2666d0-8766-4e8f-8c79-110cd2856f38/download](https://app.zenhub.com/files/133843125/be2666d0-8766-4e8f-8c79-110cd2856f38/download)
